gaeguli/gaeguli-internal.h

```diff
@@ -26,37 +26,47 @@
    \"mode\": %" G_GINT32_FORMAT " \
 }"
 
+#define PIPEWIRE_DEVICE_CLASS_STR "Stream/Output/Video"
```

Comment:
`PIPEWIRE_MEDIA_CLASS_STR` might be a better name, because `gst-device-monitor` shows the value as
```
media.class = Stream/Output/Video
```